containers/nginx-storage/delete-old-files.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash -e
+
+DIRECTORIES=(
+    /srv/storage/blender/result/
+    '/srv/storage/blender/source/'
+    /srv/storage/blender/verifier-output/
+)
+
+# function that deletes files older than `x` days, specified in the `days` argument
+function delete_old_files {
+    days="$1"
+    for directory in ${DIRECTORIES[@]}; do
+        find $directory          \
+            -type f              \
+            -mtime +$days        \
+            -delete
+    done
+}
+
+# function that deletes empty directories older than 7 days
+function delete_empty_directories {
+    for directory in ${DIRECTORIES[@]}; do
+        find $directory                   \
+            -mindepth 1                   \
+            -type d                       \
+            -mtime +7                     \
+            -exec rmdir {} 2>/dev/null \;
+    done
+}
+
+for directory in ${DIRECTORIES[@]}; do
+    if [ ! -d "$directory" ]; then
+        echo "[crond][error][disk cleanup]: Please check if the \"$directory\" directory exist."
+        exit 1
+    fi
+done
+
+free_space="$(df | grep '/srv/storage' | awk '{ print $4; }')"
+if [ -z "$free_space" ]; then
+    echo "[crond][error][disk cleanup]: There is a problem with checking free space on disk. Please check if mount point \"/srv/storage/\" exist."
+    exit 1
+fi
+
+delete_old_files 30
+
+# Convert kilobyte to gigabyte
+let free_space=$free_space/1048576
+days=29
+while (( 15 > $free_space ))
```

Comment:
please extract it to a variable, i. e. `$MINIMAL_FREE_SPACE_IN_GB`

Reply:
Done.